server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java

```diff
@@ -749,4 +738,15 @@ void cassandraMappingsSolveInconsistenciesShouldComplete() {
             .body("additionalInformation.successfulMappingsCount", is(0))
             .body("additionalInformation.errorMappingsCount", is(0));
     }
+
+    private MailboxListener.MailboxAdded createMailboxAdded() {
+        final String uuid = "6e0dd59d-660e-4d9b-b22f-0354479f47b4";
```

Comment:
We avoid using final variables

Reply:
because we don't use variable reuse but in very rare cases so it protects us from problems we should not have to deal with while adding noise.